velox/expression/tests/ExprTest.cpp

```diff
@@ -760,6 +765,36 @@ TEST_F(ExprTest, csePartialEvaluation) {
   assertEqualVectors(expected, results[1]);
 }
 
+TEST_F(ExprTest, csePartialEvaluationWithEncodings) {
+  auto data = makeRowVector(
```

Comment:
nit: would be great to add a small description here mentioning that this is a sanity check of the code path

Reply:
This reminded me that I forgot to add verification for the results. Added now.